fs/mount/fs_procfs_mount.c

```diff
@@ -101,6 +101,7 @@ struct mount_info_s
 
 /* Helpers */
 
+printflike(2, 3)
```

Comment:
move to the end

Reply:
Agreed. Will move to the end like in the other header files. In next commit.

Reply:
But it isn't good to leave the temporary change in git history.